spec/swagger_helper.rb

```diff
@@ -62,89 +62,48 @@
           },
         },
         schemas: {
-          vacancy: {
+          vacancy_request: {
```

Comment:
This is used for both create/update schema definitions. By requiring a list of particular parameters, swagger also checks that those parameters are passed on the request specs.